lib/Ohmbrewer_Temperature_Sensor.h

```diff
@@ -178,6 +184,11 @@ namespace Ohmbrewer {
              */
             Probe* _probe;
 
+            /**
```

Comment:
Correct, it doesn't affect EEPROM operation. I meant "memory" in terms of RAM at runtime. Over in Screen::displayTemps() you're setting _displayCelsius in very TemperatureSensor's Temperature object on every screen refresh.